code/game/machinery/chemHandC.dm

```diff
@@ -0,0 +1,264 @@
+
+//Heater
+
+/obj/machinery/chemheater
+	name = "directed laser heater"
+	desc = "A small stage where a container can be placed to heat it's contents with a laser."
+	icon = 'icons/obj/chemHandC.dmi'
+	icon_state = "heater"
+	icon_state_open = "heater_open"
+	density = 1
+	anchored = 1
+	machine_flags = SCREWTOGGLE | CROWDESTROY | WRENCHMOVE | FIXED2WORK
```

Comment:
Deconstructing this thing with a beaker loaded will destroy the beaker inside it, you can prevent this by using the `EJECTNOTDEL` flag

Reply:
@DamianX 
> as it is capable of reaching temperatures which could rapidly destroy any laboratory-approved container.

Reply:
That's just a joke. But I added the eject flag.